docs/04-operation-guides/troubleshooting/api-exposure/apix-11-500-server-error.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Cannot connect to a service exposed by an API Rule - 500 Internal Server Error.
+---
+
+## Symptom
+
+Having an apirule under one host URL with wildcard /* and specific /headers paths with different handlers.
```

Comment:
I would also add something about the same methods (since different methods won't have this problem)